### PR TITLE
Initial statement request functionality

### PR DESCRIPTION
**Pull request by vlad324**

### CODE REVIEW

File: contract/contracts/Statements.sol [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract Statements {
+
+    struct Request {
+        uint256 from;
```
Comment (ChmilevFA) [inline contract/contracts/Statements.sol:7]: nit: maybe for simplicity - just a calendar year?

File: contract/contracts/Statements.sol [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        uint256 to;
+        bool executed;
+        bytes cid;
```
Comment (ChmilevFA) [inline contract/contracts/Statements.sol:10]: what is it for?

**Review by ChmilevFA — APPROVED** [review]

File: contract/contracts/Statements.sol (hunk 1 above)
Comment (vlad324) [reply contract/contracts/Statements.sol:7]: what is the problem with arbitrary `from` `to`?

File: contract/contracts/Statements.sol (hunk 2 above)
Comment (vlad324) [reply contract/contracts/Statements.sol:10]: cid is IPFS identifier of the file
https://docs.ipfs.io/concepts/content-addressing/ 
so once statements file generated BE would post to IPFS and submit cid on chain for the user to download

File: contract/contracts/Statements.sol (hunk 1 above)
Comment (ChmilevFA) [reply contract/contracts/Statements.sol:7]: nothing wrong, just thoughts that probably easier for client to set the input + naturally we could just collect fees per year (per request). not important atm

**Pull request merged**